packages/clay-card/src/Body.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+export interface CardBodyProps extends React.HTMLAttributes<HTMLDivElement> {
+	children:
```

Comment:
Declare this `children` here, is it necessary? Do you have to force him like that?

Reply:
Isn't necessary